src/custom/components/Markdown/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactNode } from 'react'
+import ReactMarkdownHtml from 'react-markdown/with-html'
+import ReactMarkdown, { ReactMarkdownPropsBase } from 'react-markdown'
+import useFetchFile from 'hooks/useFetchFile'
+import { HeadingRenderer } from './renderers'
+import Page, { Title, Content } from 'components/Page'
+import styled from 'styled-components'
+import { WithClassName } from 'types'
+import { LinkScrollable, Link } from 'components/Link'
+
+interface MarkdownParams extends WithClassName {
+  contentFile: string
+  title?: ReactNode
+}
+
+export const Wrapper = styled(Page)``
+
+export function Markdown(props: { children?: string }) {
+  const { children = '' } = props
+  return <ReactMarkdown renderers={{ link: Link }}>{children}</ReactMarkdown>
+}
+
+const MarkdownContent = (props: ReactMarkdownPropsBase & { children: string }) => (
+  <ReactMarkdownHtml {...props} renderers={{ heading: HeadingRenderer, link: LinkScrollable }} allowDangerousHtml />
+)
+
+export function MarkdownPage({ contentFile, title, className }: MarkdownParams) {
+  const { error, file } = useFetchFile(contentFile)
+  return (
+    <Wrapper className={className}>
+      {title && <Title>{title}</Title>}
+      <Content>
+        {file && <MarkdownContent>{file}</MarkdownContent>}
+        {error && <MarkdownContent>{error}</MarkdownContent>}
+      </Content>
+      {/* <ScrollToTop
+        styleProps={{
+          bottom: '8.8%',
+          right: 'calc(50% - 4.6rem)',
+          background: '#9bd7c2'
+        }}
+      /> */}
```

Comment:
Any reason to keep?

Reply:
yes, i made it therefore it should stay forever

Reply:
Okok, I will upload it to the blockchain and IPFS and then im deleting it. :P

Reply:
I just did this: https://github.com/gnosis/cowswap/pull/1212/commits/bded395259fe7c10a77b7d85aa504a7eb13bf866

Reply:
Done: https://github.com/gnosis/cowswap/pull/1212/commits/1025cc93d68daad1f48ac553aada3a9387410019